app/webpack/taxa/show/components/status_tab.jsx

```diff
@@ -11,7 +11,7 @@ import {
 import _ from "lodash";
 import UserText from "../../../shared/components/user_text";
 
-const StatusTab = ( { statuses, listedTaxa, listedTaxaCount } ) => {
+const StatusTab = ( { taxonId, statuses, listedTaxa, listedTaxaCount } ) => {
```

Comment:
This new prop needs to be added to `DisagreementAlert.propTypes` at the end of the file.